examples/jsm/renderers/webgpu/WebGPUAttributes.js

```diff
@@ -27,18 +27,27 @@ class WebGPUAttributes {
 
 	}
 
-	update( attribute, isIndex = false ) {
+	update( attribute, isIndex = false, customUsage = null ) {
 
 		let data = this.buffers.get( attribute );
 
 		if ( data === undefined ) {
 
-			const usage = ( isIndex === true ) ? GPUBufferUsage.INDEX : GPUBufferUsage.VERTEX;
+			const usage = customUsage ? customUsage :
```

Comment:
How about naming `customUsage` to `usage` and then do this:
```js

if ( usage === null ) {

    usage = ( isIndex === true ) ? GPUBufferUsage.INDEX : GPUBufferUsage.VERTEX;

}

Reply:
I like the idea. Updated, thanks.

But we may need to clean up here later, for example that `isIndex` is ignored if `usage` is non-null may be a bit confusing...?